include/c/sk_types.h

```diff
@@ -1011,6 +1011,23 @@ typedef struct sk_tracememorydump_t sk_tracememorydump_t;
 typedef struct sk_runtimeeffect_t sk_runtimeeffect_t;
 typedef struct sk_runtimeeffect_uniform_t sk_runtimeeffect_uniform_t;
 
+/*
+ * Skottie Animation
+ */
+typedef struct skottie_animation_t skottie_animation_t;
+typedef struct skottie_animation_builder_t skottie_animation_builder_t;
+typedef struct skottie_resource_provider_t skottie_resource_provider_t;
+typedef struct skottie_property_observer_t skottie_property_observer_t;
+typedef struct skottie_logger_t skottie_logger_t;
+typedef struct skottie_marker_observer_t skottie_marker_observer_t;
+
+typedef struct sksg_invalidation_controller_t sksg_invalidation_controller_t;
+
+typedef enum {
+    kSkipTopLevelIsolation = 0x01,
+    kDisableTopLevelClipping = 0x02,
+} skottie_animation_renderflags_t;
+
```

Comment:
Can we convert this to all caps for consistency (with underscores) so the generator also keeps working. Also, could you add some static assets in the externals/skia/src/c/sk_enums.cpp so we catch when the reorder and randomize the list each version/